src/grafanarmadillo/find.py

```diff
@@ -53,12 +58,11 @@ def get_folder(self, name) -> FolderSearchResult:
 		else:
 			search_result = self.api.search.search_dashboards(query=name, type_="dash-folder")
 			return exactly_one(
-				list(
-					filter(
-						lambda x: x["title"] == name,
-						map(lambda sr: self.api.folder.get_folder(sr["uid"]), search_result),
-					)
-				)
+				list(filter(
+					lambda x: x["title"] == name,
+					map(lambda sr: self.api.folder.get_folder(sr["uid"]), search_result),
+				)),
+				_query_message("folder", name),
```

Comment:
Feel like i've been leaning away from `filter` seems like you loose some linting/cleanliness from them, any sense is splitting it into a couple steps, as in,
```python
folders = (self.api.folder.get_folder(entry["uid"]) for entry in search_result)
return exactly_one(
  [entry for entry in folders if entry["title"] == name],
  _query_message("folder", name),
)
```